app/src/main/res/values-night/colors.xml

```diff
@@ -117,4 +117,19 @@
     <!-- Settings -->
     <color name="settings_summary_text">#ffbbbcc5</color>
     <color name="settings_whats_new">#FFFF4AA2</color>
+
+    <!-- New dark theme -->
+    <color name="menuBackground">#342F6D</color>
+    <color name="toolbarBackground">#1D1133</color>
+    <color name="headerGradientStart">#2B2154</color>
+    <color name="headerGradientMiddle">#281D4C</color>
+    <color name="headerGradientEnd">#2A2051</color>
+    <color name="lightGrey05">@color/photonLightGrey05</color>
+    <color name="lightGrey50">@color/photonLightGrey50</color>
```

Comment:
I am wondering if we could get away with just calling photonLightGrey05/50 directly?